docs/users/subscriptions/faq/index.md

```diff
@@ -26,12 +26,15 @@ You pay for action steps that Devopness executes for you.
 
 As an action running a pipeline can have a variable number of steps, Devopness will charge you by consuming one credit for each action step executed.
```

Comment:
@pvdevs thanks, that's a good suggestion. Actually each "successful action step" consumes 1 credit. As pipelines executed by actions have a varying number of steps, different actions will consume different number of credits.

Also, please notice that Devopness does not charge credits for non successful steps, only for the the steps completed successfully.

Since this PR is already merged, would you be able to submit a pull request with your suggestions here?

Reply:
Thanks for clarifying! I’ll submit a new PR with this suggested improvement soon. 🚀